Header header
int32 class_id
string class_name
float32 confidence
float32 x_center
float32 y_center
float32 width
float32 height
float32 angle_rad # Angle in radians (YOLOv8 OBB typically outputs in radians, often -pi/2 to pi/2)